docs/css-style-guide.md

```diff
@@ -0,0 +1,92 @@
+Hypothesis CSS Style Guide
+==========================
+
+Hypothesis' CSS generally follows the principles of [Trello's CSS Style Guide](https://github.com/trello/trellisheets)
+for writing maintainable styling. In particular:
+
+ * Use simple, non-nested class selectors where possible
+ * Organize CSS into components. Unlike Trello, we use BEM-style naming (see below) and do not indent selectors
+ * Be careful with CSS preprocessor features. Unlike Trello, we use SASS but the principle
+   of using 'vanilla' CSS and being careful with preprocessor features still applies.
+   See the note about `@extends` below
+ * Use naming conventions to indicate which classes are purely presentational and which are referenced in code
+
+## File Structure
+
+The SASS for Hypothesis projects is organized into several parts:
+
+ * A 'root' file which imports all of the components used in the site, structured like this:
```

Comment:
There may be more than one "root" file. c.f. https://github.com/hypothesis/h/tree/master/h/static/styles